File: ParamConst.cpp
#include<iostream>

using namespace std;

class student{
    private:
        string fullname;
        int rollnum;
        double sempercentage;
        string collegename;
        int collegecode;
    public:
        student()
        {
            collegename = "MVGR";
            collegecode = 33;
        }
        student(string name,double percentage)
        {
            fullname = name;
            sempercentage = percentage;

        }
        ~student(){}
        void getstudentname()
        {
            cout << "Name : " << fullname  << endl ;
        }
        void getcollegename()
        {
            cout << "College : " << collegename << endl ;
        }
        void getstudentrollno()
        {
            cout << "Rollno : " << rollnum << endl ;
        }
        void getstudentmarks()
        {
            cout << "Sem percentage : " << sempercentage   << endl ;
        }
        void getcollegecode()
        {
            cout << "College code : " << collegecode << endl ;
        }
};

int main()
{
    student s1("Naga",99);

    s1.getstudentname() ; 
            
            
}

File: ParamConst.java
import java.util.Scanner;
class Student{
    String fullname;
    double sempercentage;
    String collegename;
    int collegecode;
    Student()
    {
        collegename = "MVGR";
        collegecode = 33;
    }
    Student(String name,double percentage)
    {
        fullname = name;
        sempercentage = percentage;

    }
    
   
    // protected void finalize(){
        
    //     System.out.println("object is destroyed");
    // }
    
}
public class ParamConst {
    public static void main(String[] args) {
        Scanner input  = new Scanner(System.in);

        System.out.print("Enter your name :");
        String name = input.nextLine();

        System.out.print("Enter your percentage :");
        double sem = input.nextDouble();

        Student s1 = new Student(name,sem);
        Student s2 = new Student(); 
        System.out.println("Name : "+s1.fullname);
        System.out.println("sem percentage : "+s1.sempercentage);
        System.out.println("college name : "+s2.collegename);
        System.out.println("college code : "+s2.collegecode);

        input.close();
    }
    
}
